app/templates/flat-page.hbs

```diff
@@ -0,0 +1,56 @@
+<div class="l-constrained flatpage">
+  {{#if isFastBoot}}
+    <div class=" django-content flatpage-loader--loading">
+      Loading...
+    </div>
+  {{else if flatPage.content}}
+    <div class="django-content">
+      {{#if (eq flatPage.templateName 'flatpages/graphic_feature_plus.html')}}
+        <div class="article-description">
+          <link rel="stylesheet" type="text/css" href="/static/css/bootstrap.min.css">
+          {{active-html html=flatPage.content }}
+        </div>
+      {{else}}
+        <div class="graphic-default">
+          <div class="grid_8 main l-col2of3">
+            <div class="page-titlegroup">
+              <h1 class="h1 h1--extrabold h1--mq-small">{{flatPage.title}}</h1>
+            </div>
+            <div class="article-tools clearfix">
+              <div class="share-tools clearfix">
+                <ul class="sharelinks clearfix">
+                  <li class="fa-hover facebook">
+                    <a title="Share on Facebook" class="fa-hover-link" href="https://www.facebook.com/sharer/sharer.php?u=http%3A%2F%2Fwww.wqxr.org%2Fabout%2F" data-share-via="facebook" data-story-title="About+WQXR" data-show-name="About+WQXR">
```

Comment:
Looks like there is some static content left over from a possible view source copy / paste in this section. (See share href and data-attrs.)